tests/acceptance/test_sections_and_flows.py

```diff
@@ -0,0 +1,503 @@
+    # Iteratively remove all rows if any
+    for _ in range(50):
+        names = _table_filenames(page)
+        if not names:
+            break
+        name = names[0]
+        try:
+            _click_table_cell_with_text(page, name)
+            page.get_by_text(
+                rm.get(AddVideoKeys.BUTTON_REMOVE_VIDEOS), exact=True
+            ).click()
+            # wait until it's gone
+            deadline = time.time() + ACCEPTANCE_TEST_WAIT_TIMEOUT
+            while time.time() < deadline:
+                if name not in _table_filenames(page):
+                    break
+                time.sleep(0.05)
+        except Exception:
+            # Best-effort cleanup only
+            break
+
+
+def _add_video_via_picker(page: Page, rm: ResourceManager, path: Path) -> None:
+    page.get_by_text(rm.get(AddVideoKeys.BUTTON_ADD_VIDEOS), exact=True).click()
+    ui_path = path.relative_to(file_picker_directory())
+    # Double-click each path segment within the file picker grid (ag-grid)
+    for part in ui_path.parts:
+        _open_part(page, part)
+
+
+def _open_part(page: Page, part: str) -> None:
+    deadline = time.time() + ACCEPTANCE_TEST_WAIT_TIMEOUT
+    last_err: Exception | None = None
+    while time.time() < deadline:
+        try:
+            cell = page.locator(".ag-cell-value", has_text=part).first
+            cell.wait_for(state="visible", timeout=1000)
+            cell.dblclick()
+            return
+        except Exception as e:
+            last_cell = page.locator(".ag-cell-value").last
+            try:
+                last_cell.scroll_into_view_if_needed()
+            except Exception:
+                pass
+            last_err = e
+    if last_err:
+        raise last_err
+    raise AssertionError(f"Could not find table cell with text: {part}")
+
+
+class TestAddLineSectionWithDialog:
+
+    @pytest.mark.skip(reason="only works in headed right now")
+    @pytest.mark.timeout(300)
+    @pytest.mark.playwright
+    @pytest.mark.usefixtures("external_app")
+    def test_add_line_section_with_dialog(
+        self,
+        page: Page,
+        external_app: NiceGUITestServer,
+        resource_manager: ResourceManager,
+    ) -> None:
+        """Playwright: Add a line section via canvas interaction and confirm via dialog.
+
+        Steps:
+        - Open main page, switch to Videos tab
+        - Add a video via in-app file picker UI and select it to render a frame
+        - Switch to Sections tab, activate "Add line", click on the interactive image
+        - Enter a name in the dialog and apply
+        - Verify the section name appears on the page
+
+        Prerequisites: pytest-playwright installed and browsers set up.
+        """
+        base_url = getattr(
+            external_app, "base_url", "http://127.0.0.1:8080"
+        )  # fallback
+        page.goto(base_url + ENDPOINT_MAIN_PAGE)
+
+        # Switch to Videos tab
+        page.get_by_text(
```

Comment:
Can we open the videos tab using a marker (`MARKER_VIDEO_TAB`)? Or at least rename the `TAB_TWO` constant to something related to `VIDEO`.